app/models/job.rb

```diff
@@ -171,7 +171,7 @@ def self.check_jobs_for_timeout
         .where("state != 'finished' and (state != 'waiting_to_start' or dispatch_status = 'active')")
         .where("zone is null or zone = ?", MiqServer.my_zone)
         .each do |job|
-          next unless job.updated_on < job.current_job_timeout.seconds.ago
+          next unless job.updated_on < job.current_job_timeout(timeout_adjustment(job)).seconds.ago
```

Comment:
It looks like `current_job_timeout` always returns `DEFAULT_TIMEOUT`, so how does this adjust the timeout?